appveyor.yml

```diff
@@ -29,7 +29,7 @@
 
 version: 1.0.{build}
 build_cloud: libcc-20
-image: vs2019bt-16.4.0
+image: vs2019-16.3-10.0.18362
```

Comment:
@jkleinsc lets handle this revert in a separate PR after merging this roll, chromium was looking for VS version in ci that was not available.

Reply:
@deepak1556 that is strange -- master works with visual studio build tools.  What was the error?

Reply:
Yeah that was the weird part, I didn't investigate further , just reverted to avoid blocking the roll https://ci.appveyor.com/project/electron-bot/electron-ljo26/builds/30148279

Reply:
@deepak1556 it looks like this was the change that caused the breakage:
https://chromium.googlesource.com/chromium/src/+/dad9e0f52b911e561bed2073be39fa3540797d80
Let me look at it quick